core/templates/pages/exploration-editor-page/editor-tab/templates/modal-templates/customize-interaction-modal.controller.spec.ts

```diff
@@ -111,6 +111,34 @@ describe('Customize Interaction Modal Controller', function() {
       });
     }));
 
+    it('should properly detect a SubtitledUnicode schema', () => {
```

Comment:
Make sure to check the reverse, i.e. that non-subtitled-unicode schemas (that are close to a subtitled unicode one) are correctly identified as false. Ditto for html.

Reply:
Done